### PR TITLE
Feature/e460 styling

### PR DESCRIPTION
**Pull request by dudiq**

### CODE REVIEW

**Comment by dudiq** [comment]
@lamartire ready for review

File: source/index.html.md.erb [hunk 1]
```diff
@@ -15,5 +15,5 @@ search: true
 <%= partial "includes/_quick-start.md.erb" %>
 <%= partial "includes/_concepts.md.erb" %>
 <%= partial "includes/_authentication.md.erb" %>
-<%= partial "includes/connect/_connect.md.erb" %>
```
Comment (kazurus) [inline source/index.html.md.erb]: please, keep old version

File: source/includes/server-api/_scopes.md.erb [hunk 2]
```diff
@@ -14,7 +14,8 @@ Returns scopes granted by user for particular application.
     "documents:image:read",
     "wallet:address:read",
     "wallet:accounts:read",
-    "user:email:read"
+    "user:email:read",
+    "user:data:read"
```
Comment (kazurus) [inline source/includes/server-api/_scopes.md.erb]: add to concepts scopes as well

File: source/includes/server-api/_personal-data.md.erb [hunk 3]
```diff
@@ -21,6 +21,7 @@ Returns user's personal data fields.
 |`firstName` (*string*)                      | First name        |
 |`lastName` (*string*)                       | Last name         |
 |`dateOfBirth`      (*number*)               | Timestamp, UTC tz |
+|`verified`         (*boolean*)              | Is verified       |
```
Comment (kazurus) [inline source/includes/server-api/_personal-data.md.erb]: to example as well

File: source/includes/server-api/_user.md.erb [hunk 4]
```diff
@@ -53,6 +53,7 @@ Generic user information. By default, only public information is shared without
 |`status` (*string*, [*enumerable*](#status))|Verification status|
 |`country` (*string*)                        |Country code       |
 |`number` (*string*)                         |Phone number       |
+|`main` (*boolean*)                          |Is main phone      |
```
Comment (kazurus) [inline source/includes/server-api/_user.md.erb]: to example as well

File: source/includes/server-api/_address.md.erb [hunk 5]
```diff
@@ -1,12 +1,16 @@
 ## User's address
 
-Scope: `user:address:read`
+### Scopes
```
Comment (kazurus) [inline source/includes/server-api/_address.md.erb]: scopes should be in place such other

**Pull request merged**